src/services/findAllReferences.ts

```diff
@@ -769,7 +769,7 @@ namespace ts.FindAllReferences {
                         // At `module.exports = ...`, reference node is `module`
                         const node = isBinaryExpression(decl) && isPropertyAccessExpression(decl.left) ? decl.left.expression :
                             isExportAssignment(decl) ? Debug.checkDefined(findChildOfKind(decl, SyntaxKind.ExportKeyword, sourceFile)) :
-                            getNameOfDeclaration(decl) || decl;
+                                getNameOfDeclaration(decl) || decl;

```

Comment:
Can we undo all the auto formatting changes?

```suggestion
                            getNameOfDeclaration(decl) || decl;
```